packages/tools/src/tools/annotation/ArrowAnnotateTool.ts

```diff
@@ -181,20 +183,21 @@ class ArrowAnnotateTool extends AnnotationTool {
           points: [<Types.Point3>[...worldPos], <Types.Point3>[...worldPos]],
           activeHandleIndex: null,
           arrowFirst,
-          textBox: {
-            hasMoved: false,
-            worldPosition: <Types.Point3>[0, 0, 0],
-            worldBoundingBox: {
-              topLeft: <Types.Point3>[0, 0, 0],
-              topRight: <Types.Point3>[0, 0, 0],
-              bottomLeft: <Types.Point3>[0, 0, 0],
-              bottomRight: <Types.Point3>[0, 0, 0],
-            },
-          },
         },
-        label: '',
       },
     };
+    if (!this.configuration.renderTextBox) {
```

Comment:
This should really be initialized, because the app config can switch renderTextBox on or off, and if it's not initialized, it will throw an error since the data is missing.